public/src/managers/CourseManager.ts

```diff
@@ -362,6 +388,27 @@ export class CourseManager {
         }
     }
 
+    /**
+     * Add IStudentSubmissions to an IUserCourse
+     * @param group The group
+     * @param groupCourse The group course
+     */
+    private async fillLinksGroup(group: ICourseGroup, groupCourse: IGroupCourse): Promise<void> {
```

Comment:
The first argument `group` is of type `ICourseGroup`. This isn't clear in the documentation and in the naming of the argument.

Reply:
I agree that it is confusing (have a TODO for renaming that interface) but lack of a better name, and I saw that for group the interface is named ICourseGroup instead of just IGroup, which would be similar to the IUser naming.
I'm thinking of refactoring IGroupCourse to IGroup, and let ICourseGroup stay the same, which is the same naming as ICourseUser.